src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
This code is hard to read by humans, making difficult to maintain in the future, if we don't have a performance gain and requirement it's better to use meaningful code (clean code). 
Also we have duplicated code, so a better approach would be.
```suggestion
 // In this case 'm' could be used since it's a convention all around the world, but always prefer variables like this to meaningfull variables and avoid maginc number and strings ( Magic number/string: 
const male = 'm'
 function isMale(person) {
 person.sex === male
 }
 function diedInCentury( person, century ) {
  return Math.ceil(person.died / 100) === century)
 }
 const men = century
    ? people.filter((person) => isMale(person) && diedInCentury( person, century )
    : people.filter((p) => isMale(person));
```
Another problem here is the way you are dealing with 'century' since its a number if you use ternary conditional '?' you are going to get the wrong results if the value is 0, because it validates against falsy values and 0 is a falsy value. 
Please fix it and validate the logic against 0.

https://mate-academy.github.io/style-guides/javascript.html#231avoid-single-letter-names-be-descriptive-with-your-naming